content/applications/hr/lunch/vendors.rst

```diff
@@ -0,0 +1,153 @@
+=======
+Vendors
+=======
+
+Before :doc:`products can be added <products>` to the *Lunch* app, the restaurants that provide the
+food **must** be configured.
+
+To add a new vendor, first navigate to :menuselection:`Lunch app --> Configuration --> Vendors`.
+Here, all currently configured vendors for the *Lunch* app appear in a default Kanban view. To
+change to a list view, click the :icon:`oi-view-list` :guilabel:`(list)` icon in the top-right
+corner.
+
+.. note::
+   No vendors are preconfigured in the *Lunch* app, so all vendors **must** be added to the
+   database.
+
+To add a new vendor, click the :guilabel:`New` button in the top-left corner, and a new lunch
+supplier form loads.
+
+Fill out the following fields on the form.
```

Comment:
Entirerly optional, but I think it would help the flow of the doc to link to each of the sections in the vendor form :)

(You will need to add custom anchors to the sections missing them)
```suggestion
Fill out the following sections on the vendor form:

- :ref:`Vendor information <lunch/vendor-info>`
- :ref:`Availability <lunch/availability>`
- :ref:`Orders <lunch/orders>`
- :ref:`Extras <lunch/extras>`
```

Reply:
I LOVE this idea!!